build.sbt

```diff
@@ -61,10 +61,11 @@ libraryDependencies ++= Seq(
   "org.influxdb" % "influxdb-java" % "2.14",
   "org.apache.kafka" %% "kafka" % "2.2.0" % "provided",
   "za.co.absa" % "abris_2.11" % "3.1.1"  % "provided" excludeAll(excludeAvro, excludeSpark),
-  "com.uber.hoodie" % "hoodie-spark" % "0.4.7" % "provided" excludeAll(excludeHoodieTimeLineService),
-  "com.uber.hoodie" % "hoodie-common" % "0.4.7" % "provided" excludeAll(excludeHoodieTimeLineService),
+  "org.apache.hudi" % "hudi-spark" % "0.5.0-incubating" % "provided" excludeAll(excludeHoodieTimeLineService),
```

Comment:
0.5.1

Reply:
Also please remove exclusion and excludeHoodieTimeLineService from above, no longer needed